charts/datadog/values.yaml

```
@@ -319,6 +319,9 @@ datadog:
     ## ref: https://docs.datadoghq.com/infrastructure/process/?tab=linuxwindows#process-arguments-scrubbing
     stripProcessArguments: false
 
+    # datadog.processAgent.processDiscovery -- Set this to false to disable autodiscovery of integrations
```

we can update the description to reflect the change made on the default value.

```suggestion
    # datadog.processAgent.processDiscovery -- Set this to true to enable autodiscovery of integrations
```

---

Sure, I'm actually just going to make it generic since we're going to change the default to true in 7.33

---

Should be fixed now, let me know if you don't think we should use the generic version